src/app/(guest)/exams/[id]/review/page.tsx

```diff
@@ -75,8 +72,8 @@ const ReviewPage: React.FC<ExamPageProps> = ({ params }) => {
 
   return (
     <section className="h-[88vh] flex flex-col items-center">
-      <p className="text-xl font-bold uppercase mt-10 ml-5 text-center">
-        Exame de <span className="text-primary">{subject}</span>
+      <p className="text-xl font-bold uppercase mt-10 ml-5 text-center px-4">
```

Comment:
Only show this if examResult is truthy, because if the examResult is false, then you gonna show content that does not make sense without the exam
```suggestion
{ !!examResult ?
      <p className="text-xl font-bold uppercase mt-10 ml-5 text-center px-4">
              Exame de <span className="text-primary">{examResult.subject}</span>
      </p>
      : null
      }
```

Reply:
to enhance ux just wrap the name inside the conditional and show some placeholder component (e.g. skeleton)

Reply:
Good sugestion, we need to create more skeleton loadings across the whole application